e2e_tests/run.py

```
@@ -0,0 +1,101 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)